### DISCUSSION TITLE
how do i solve this ?

### DISCUSSION THREAD

**Discussion by ayesparshh** [Cyfrin/foundry-full-course-cu#2064] [General]
![Screenshot 2024-07-20 143709](https://github.com/user-attachments/assets/28748d6d-d6f8-49d9-9c7b-7af2dcf3dc6e)
Like I know how to call a method on my deployed contract but how to call a method on someone else deployed contract?
please help

**Comment by ayesparshh** [comment]
solved

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.
